Software/Python/easygopigo3.py

```diff
@@ -28,8 +28,6 @@
 except:
```

Comment:
This is the second one I'm worried about.  I will defer to @CleoQc  on this one though.  We might also have @mattallen37  look at it.